FabNESO/tasks.py

```
@@ -21,10 +27,14 @@
     from types import SimpleNamespace
 
     fab = SimpleNamespace(task=lambda f: f, load_plugin_env_vars=lambda _: lambda f: f)
+    template = SimpleNamespace(
+        task=lambda f: f, load_plugin_env_vars=lambda _: lambda f: f
+    )
```

Not sure this should be necessary as it looks like `template` is only used within function definitions? The slightly hacky approach I used for the `fab` module was only required there as we use the `task` and `load_plugin_env_vars` functions in the `fab` module as decorators, which will be evaluated at import time, so if these names are missing due to `fabsim` not being importable we will get a `NameError`. For `template` as this is I think not accessed at import time, there shouldn't be a need to create a placeholder (and if `fabsim` is not importable at run time the `_check_fab_module_imported` function should raise an error before we get to `template` call).

---

I'm not sure if it did raise an error at some point (I'm not sure why I'd have thought to do the same thing otherwise...), but it doesn't seem to anymore, so I've taken out this part from the most recent version,